.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# PR Approval Steps
+
+## For Requester
+
+1. Description
+    - [ ] Check the PR title and description for clarity. It should describe the changes made and the reason behind them.
+    - [ ] Ensure that the PR follows the contribution guidelines, if applicable.
+2. Security requirements
+    - [ ] Ensure that a Pull Request (PR) does not expose passwords and other sensitive information by using git-secrets and upload relevant evidence: https://github.com/awslabs/git-secrets
+    - [ ] Ensure commit has GitHub Commit Signature
+3. Automatic checks in GitHub Workflow
+    - [ ] collab-check : Check if the owner of PR belongs is collaborator of this repo or not
```

Comment:
why PR owner is collaborator? anyone can be collaborator?

Reply:
All github users within our github team are collaborators